src/python/examples/CPUWeight

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# vim:sw=4:ts=4:et
+from hirte import Hirte
```

Comment:
I would leave the examples out of the `src/python/` (even out of `src` itself) if we can. They aren't really part of the source code and we don't want to install them (which I suspect the current `setup.py` does).